.ci/run-regression.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` please for better compatibility with Nix.

Reply:
Fixed.